hieroplatform/src/main/java/org/hiero/table/Schema.java

```diff
@@ -86,7 +86,7 @@ public int getColumnCount() {
         return this.columns.size();
     }
 
-    public Iterable<String> getColumnNames() {
+    public Set<String> getColumnNames() {
```

Comment:
I think that Iterable is more general than Set. Do you have any strong reason for using Set?